R/update-news.R

```diff
@@ -1,6 +1,6 @@
 # File editing ------
 
-update_news_impl <- function(commits, which) {
+update_news_impl <- function(commits, which, fledgeling = NULL) {
```

Comment:
Would the following be an anti-pattern, violating https://design.tidyverse.org/def-short.html and similar?

```suggestion
update_news_impl <- function(commits, which, fledgeling = read_fledgeling()) {
```